app/views/support/WeeklyPromotion.scala

```diff
@@ -7,63 +7,57 @@ import com.gu.memsub.promo.{PromoCode, WeeklyLandingPage}
 import com.gu.memsub.subsv2.{CatalogPlan, Catalog => SubsCatalog}
 import com.netaporter.uri.Uri
 import com.netaporter.uri.dsl._
+import model.GuardianWeeklyZones
 import views.support.Pricing._
 
 object WeeklyPromotion {
   private val UK = CountryGroup.UK.countries.toSet
   private val UKdomestic = Set(CountryGroup.C.UK)
   private val UKoverseas = UK diff UKdomestic
-  private val US = CountryGroup.US.countries.toSet
-  private val AU = CountryGroup.Australia.countries.toSet
-  private val NZ = CountryGroup.NewZealand.countries.toSet
-  private val EU = CountryGroup.Europe.countries.toSet
-  private val CA = CountryGroup.Canada.countries.toSet
+
   private val allCountries = CountryGroup.allGroups.flatMap(_.countries).toSet
-  private val ZONEC = allCountries.diff(UK union US)
+
   case class DiscountedPlan(currency: Currency, pretty: String, headline: String, period: String, url: Uri, discounted: Boolean)
 
-  case class DiscountedRegion(title: String, description: String, countries: Set[Country], discountedPlans: List[DiscountedPlan])
+  case class DiscountedRegion(title: String, description: String, discountedPlans: List[DiscountedPlan])
 
   def validRegionsForPromotion(promotion: Option[PromoWithWeeklyLandingPage], promoCode: Option[PromoCode], requestCountry: Country)(implicit catalog: SubsCatalog): Seq[DiscountedRegion] = {
     val promotionCountries = promotion.map(_.appliesTo.countries).getOrElse(allCountries)
-    val regionForZoneCCountry: Seq[DiscountedRegion] = {
-      if (ZONEC.contains(requestCountry)) {
+
+    val separateRegionIfRequestCountryIsRestOfWorld: Seq[DiscountedRegion] = {
+      if (PlanPicker.isInRestOfWorldOrZoneC(requestCountry: Country)) {
         val currency = CountryGroup.byCountryCode(requestCountry.alpha2).map(_.currency).getOrElse(Currency.USD)
         Seq(DiscountedRegion(
           title = requestCountry.name,
           description = "Posted to you by air mail",
-          countries = Set(requestCountry),
-          discountedPlans = plansForPromotion(promotion, promoCode, Set(requestCountry), catalog.weekly.zoneC.plans,currency)
+          discountedPlans = plansForPromotion(promotion, promoCode, PlanPicker.plans(requestCountry),currency)
         ))
       } else {
         Seq()
       }
     }
-    val rowWithoutCountry: Seq[DiscountedRegion] = Seq(DiscountedRegion(
-        title = "Rest of the world",
-        description = "Posted to you by air mail",
-        countries = ZONEC - requestCountry,
-        discountedPlans = plansForPromotion(promotion, promoCode, ZONEC - requestCountry, catalog.weekly.zoneC.plans,Currency.USD)
-      ))
+
+    val restOfWorldRegion = Seq(DiscountedRegion(
+      title = "Rest of the world",
```

Comment:
One for a follow up PR, but I wonder if 'Rest of the world' is a user-friendly label for this?